spec/vcr_setup.rb

```diff
@@ -10,7 +10,7 @@
   c.filter_sensitive_data('*100EYES') { ENV.fetch('THREEMARB_API_IDENTITY', nil) }
   c.filter_sensitive_data('THREEMARB_API_SECRET') { ENV.fetch('THREEMARB_API_SECRET', nil) }
   c.filter_sensitive_data('THREEMARB_PRIVATE') { ENV.fetch('THREEMARB_PRIVATE', nil) }
-  c.filter_sensitive_data('SIGNAL_SERVER_PHONE_NUMBER') { ENV.fetch('SIGNAL_SERVER_PHONE_NUMBER', nil) }
+  c.filter_sensitive_data('SIGNAL_SERVER_PHONE_NUMBER') { Setting.signal_server_phone_number }
```

Comment:
should we apply the same change to the other filters?

Reply:
I think it would be nice for it to be consistent and I guess it would make sense to apply the change in the same PR. Alternatively, I went with another approach in[ Improve setting channels logic #1818](https://github.com/tactilenews/100eyes/pull/1818) that would also fix this problem and I think I actually prefer it.

I think we do not want to be loading values from our local .env files. Having a dedicated `.env.local.test` would allow us to make sure we are not loading our local .env files if they exist and would then mean we would be testing the same between our machines and ci.

Reply:
If you have good experience with that gem and see no issues, I would also go with it but see comment https://github.com/tactilenews/100eyes/pull/1818#pullrequestreview-2005405247

Reply:
said that I'm then fine to have this PR merged as it is.